lib/internal/Magento/Framework/GraphQl/Query/Fields.php

```diff
@@ -62,4 +67,24 @@ public function getFieldsUsedInQuery()
     {
         return $this->fieldsUsedInQuery;
     }
+
+    /**
+     * Extract and return list of all used fields in GraphQL query's variables
+     *
+     * @param array $variables
+     *
+     * @return string[]
+     */
+    private function getVariables(array $variables): array
```

Comment:
```suggestion
    private function formatVariables(array $variables): array
```
or maybe `extractVariables`